src/openzaak/components/zaken/tests/test_relevantezaakrelatie.py

```diff
@@ -72,6 +72,48 @@ def test_create_external_relevante_andere_zaak(self):
 
         self.assertEqual(response.status_code, status.HTTP_201_CREATED, response.data)
 
+    def test_create_external_relevante_andere_zaak_without_setting_gerelateerde_zaak_typen(
+        self,
+    ):
+        zaaktype = ZaakTypeFactory.create(concept=False)
+        zaaktype_url = f"http://testserver.com{reverse(zaaktype)}"
+        zaak_external = (
+            "https://externe.zaken.nl/api/v1/zaken/a620b183-d898-4576-ae94-3f21d43cc576"
+        )
+
+        # gerelateerde_zaak_typen are not set.
+        self.assertEqual(zaaktype.zaaktypenrelaties.count(), 0)
+
+        with requests_mock.Mocker() as m:
+            mock_zrc_oas_get(m)
+            m.get(zaak_external, json=get_zaak_response(zaak_external, zaaktype_url))
+
+            response = self.client.post(
+                self.list_url,
+                {
+                    "zaaktype": zaaktype_url,
```

Comment:
I guess technically this is correct, but the same zaaktype is used for both zaken. Could you modify both tests to have two different zaaktypen and verify that that works?